itest/rust/src/transform2d_test.rs

```diff
@@ -20,54 +21,47 @@ pub(crate) fn run() -> bool {
-        Vector2::is_equal_approx(a, b)
-    });
-    assert_inner_equivalent!(InnerTransform2D, TEST_TRANSFORM.(skew,get_skew)(), |a, b| {
-        is_equal_approx(a, b as f32)
-    });
-    assert_inner_equivalent!(
-        InnerTransform2D,
-        TEST_TRANSFORM.orthonormalized(),
-        |a, b| Transform2D::is_equal_approx(&a, &b)
-    );
-    assert_inner_equivalent!(InnerTransform2D, TEST_TRANSFORM.rotated(1.0), |a, b| {
-        Transform2D::is_equal_approx(&a, &b)
-    });
-    assert_inner_equivalent!(
-        InnerTransform2D,
-        TEST_TRANSFORM.rotated_local(1.0),
-        |a, b| Transform2D::is_equal_approx(&a, &b)
-    );
-    assert_inner_equivalent!(
-        InnerTransform2D,
-        TEST_TRANSFORM.scaled(Vector2::new(1.0, 2.0)),
-        |a, b| Transform2D::is_equal_approx(&a, &b)
-    );
-    assert_inner_equivalent!(
-        InnerTransform2D,
-        TEST_TRANSFORM.scaled_local(Vector2::new(1.0, 2.0)),
-        |a, b| Transform2D::is_equal_approx(&a, &b)
-    );
-    assert_inner_equivalent!(
-        InnerTransform2D,
-        TEST_TRANSFORM.translated(Vector2::new(1.0, 2.0)),
-        |a, b| Transform2D::is_equal_approx(&a, &b)
+    let inner = InnerTransform2D::from_outer(&TEST_TRANSFORM);
+    let outer = TEST_TRANSFORM;
+    let vec = Vector2::new(1.0, 2.0);
+
+    #[rustfmt::skip]
+    let mappings_transform = [
+        ("affine_inverse",   inner.affine_inverse(),                             outer.affine_inverse()                            ),
+        ("orthonormalized",  inner.orthonormalized(),                            outer.orthonormalized()                           ),
+        ("rotated",          inner.rotated(1.0),                                 outer.rotated(1.0)                                ),
+        ("rotated_local",    inner.rotated_local(1.0),                           outer.rotated_local(1.0)                          ),
+        ("scaled",           inner.scaled(vec),                                  outer.scaled(vec)                                 ),
+        ("scaled_local",     inner.scaled_local(vec),                            outer.scaled_local(vec)                           ),
+        ("translated",       inner.translated(vec),                              outer.translated(vec)                             ),
+        ("translated_local", inner.translated_local(vec),                        outer.translated_local(vec)                       ),
+        ("interpolate_with", inner.interpolate_with(Transform2D::IDENTITY, 0.5), outer.interpolate_with(Transform2D::IDENTITY, 0.5))
+    ];
+    for (name, inner, outer) in mappings_transform {
+        assert_eq_approx!(
+            &inner,
+            &outer,
+            Transform2D::is_equal_approx,
+            "function: {name}\n"
+        );
+    }
+
+    assert_eq_approx!(
+        inner.get_rotation(),
+        outer.rotation(),
+        |a, b| is_equal_approx(a as f32, b),
+        "function = get_rotation\n"
```

Comment:
maybe also use `function: get_rotation` (colon instead of equals, like above)

Reply:
oh woops